spec/apis/v1/submission_comments_api_spec.rb

```diff
@@ -22,6 +22,23 @@
 
 describe 'Submissions Comment API', type: :request do
 
+  def submission_with_comment
+    course_with_student(:active_all => true)
+    teacher_in_course(:course => @course, :active_all => true)
+    @assignment = @course.assignments.create!(
+      :title => "Test Assignment",
+      :description => "public stuff"
+    )
+    @student = @course.students.first
+    @submission = @assignment.submissions.find_or_create_by(
+      :user => @student
+    )
+    @comment = @submission.submission_comments.create!(
+      :comment => "Hello world!",
+      :author => @user
```

Comment:
I think `@teacher` is set as well, and if we use that instead of `@user`, it would be more clear who owns this comment.